lib/actions-oclif/device/rm.ts

```diff
@@ -72,12 +74,19 @@ export default class DeviceRmCmd extends Command {
 		const patterns = await import('../../utils/patterns');
 
 		// Confirm
-		await patterns.confirm(
-			options.yes,
-			'Are you sure you want to delete the device?',
-		);
+		for (const uuid of params.uuid.split(',')) {
+			try {
+				await patterns.confirm(
```

Comment:
hmm I think it would be better to ask this confirmation question only once for all devices, before the `for` loop. For example:
```
Are you sure you want to delete 135 devices?
```
An option would be to include the device UUID in the confirmation message if only one device is given, and otherwise mention only the total count of devices.

Reply:
Agreed. I was thinking if the user didn't like answering them all then they could use the `--yes` argument but I think this is a better option.